upload_liz_genomes.py

```diff
@@ -1,148 +1,70 @@
-"MTB2.1.fastq.gz",
-"MTB9.1.fastq.gz",
-"MTB17.1.fastq.gz",
-"MTB25.1.fastq.gz",
-"MTB32.1.fastq.gz",
-"MTB39.1.fastq.gz",
-"MTB47.1.fastq.gz",
-"MTB2.2.fastq.gz",
-"MTB9.2.fastq.gz",
-"MTB17.2.fastq.gz",
-"MTB25.2.fastq.gz",
-"MTB32.2.fastq.gz",
-"MTB39.2.fastq.gz",
-"MTB47.2.fastq.gz",
-"MTB3.1.fastq.gz",
-"MTB10.1.fastq.gz",
-"MTB18.1.fastq.gz",
-"MTB26.1.fastq.gz",
-"MTB33.1.fastq.gz",
-"MTB41.1.fastq.gz",
-"MTB48.1.fastq.gz",
-"MTB3.2.fastq.gz",
-"MTB10.2.fastq.gz",
-"MTB18.2.fastq.gz",
-"MTB26.2.fastq.gz",
-"MTB33.2.fastq.gz",
-"MTB41.2.fastq.gz",
-"MTB48.2.fastq.gz",
-"MTB4.1.fastq.gz",
-"MTB11.1.fastq.gz",
-"MTB19.1.fastq.gz",
-"MTB27.1.fastq.gz",
-"MTB34.1.fastq.gz",
-"MTB42.1.fastq.gz",
-"MTB49.1.fastq.gz",
-"MTB4.2.fastq.gz",
-"MTB11.2.fastq.gz",
-"MTB19.2.fastq.gz",
-"MTB27.2.fastq.gz",
-"MTB34.2.fastq.gz",
-"MTB42_2.fastq.gz",
-"MTB49.2.fastq.gz",
-"MTB5.1.fastq.gz",
-"MTB13.1.fastq.gz",
-"MTB20.1.fastq.gz",
-"MTB28.1.fastq.gz",
-"MTB35.1.fastq.gz",
-"MTB43.1.fastq.gz",
-"MTB50.1.fastq.gz",
-"MTB5.2.fastq.gz",
-"MTB13.2.fastq.gz",
-"MTB20.2.fastq.gz",
-"MTB28.2.fastq.gz",
-"MTB35.2.fastq.gz",
-"MTB43.2.fastq.gz",
-"MTB50.2.fastq.gz",
-"MTB6.1.fastq.gz",
-"MTB14.1.fastq.gz",
-"MTB21.1.fastq.gz",
-"MTB29.1.fastq.gz",
-"MTB36.1.fastq.gz",
-"MTB44.1.fastq.gz",
-"MTB51.1.fastq.gz",
-"MTB6.2.fastq.gz",
-"MTB14.2.fastq.gz",
-"MTB21.2.fastq.gz",
-"MTB29.2.fastq.gz",
-"MTB36.2.fastq.gz",
-"MTB44.2.fastq.gz",
-"MTB51.2.fastq.gz",
-"MTB7.1.fastq.gz",
-"MTB15.1.fastq.gz",
-"MTB22.1.fastq.gz",
-"MTB30.1.fastq.gz",
-"MTB37.1.fastq.gz",
-"MTB45.1.fastq.gz",
-"MTB7.2.fastq.gz",
-"MTB15.2.fastq.gz",
-"MTB22.2.fastq.gz",
-"MTB30.2.fastq.gz",
-"MTB37.2.fastq.gz",
-"MTB45.2.fastq.gz"
-    ]
+parser.add_argument("-f","-folder",action='store', help="Folder containing all files",required = True)
+parser.add_argument("-e","-extension",action="store", help= "File extension from files",required = False, default="fastq.gz")
+parser.add_argument("-u","-username",action='store', help="NCBI - Username",required = True)
+parser.add_argument("-p","-password",action='store', help= "NCBI - Password",required = True)
+
+args = parser.parse_args()
+
+file_ext = args.e
+foldername = args.f
+username = args.u
+password = args.p
+
+
+
+# 2. List of names of genomes to be uploaded
+
+genome_list = [ foldername+"/"+i for i in os.listdir(foldername) if file_ext in i]
 
 
 """
-2. Update the `username` and `password` 
+3. Update the `username` and `password` 
 """
 
+
 ftp = ftplib.FTP("ftp-private.ncbi.nlm.nih.gov", username, password)
 
 
 """
-3. Create a directory using FTP, once you've logged in and you have your own address like `emilyncosta_gmail.com_xxxxx` and create a directory where you'll upload submission `africanum`
+4. Create a directory using FTP, once you've logged in and you have your own address like `emilyncosta_gmail.com_xxxxx` and create a directory where you'll upload submission `africanum`
 """
 
 ftp.cwd("uploads/lizaspinasse_gmail.com_ojGYyyaI")
```

Comment:
I think we can also take this folder address via args as well. Would you mind adding the `-d` option, I [suggested](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) to avoid hardcoding stuff.

Reply:
Thank you, I'll implement that and make the last commit